captum/metrics/_core/sensitivity.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python3
+
+from copy import deepcopy
+
+import torch
+
+from ..._utils.common import (
+    _expand_and_update_additional_forward_args,
+    _expand_and_update_baselines,
+    _expand_and_update_target,
+    _format_input,
+    _format_tensor_into_tuples,
+)
+from .._utils.batching import _divide_and_aggregate_metrics
+
+
+def default_perturb_func(inputs, perturb_radius=0.02):
```

Comment:
Good point! To keep it simple, I think that at this point I prefer to leaving the default function as infinity. In the future if needed we can add and expose all those options as well. Also because default_perturb_func isn't exposed in the API otherwise we have to expose all of them in the API.